taxonomy-editor-frontend/src/pages/gotoproject/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { Typography, Box, Grid } from "@mui/material";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import useFetch from "../../components/useFetch";
+import { API_URL } from "../../constants";
+import MaterialTable from '@material-table/core';
+import EditIcon from '@mui/icons-material/Edit';
+import { useEffect } from "react";
+import { toSnakeCase, toTitleCase } from "../editentry/interConvertNames";
+
+const GotoProject = () => {
+    const { data: incomingData, isPending, isError, isSuccess, errorMessage } = useFetch(`${API_URL}projects`);
+    const [projectData, setProjectData] = useState([]);
+    const navigate = useNavigate();
+
+    useEffect(() => { 
+        const renderedProjects = [];
+        if (incomingData) {
+            console.log(incomingData);
+            incomingData.forEach((element) => {
+                const projectNode = element[0];
+                renderedProjects.push({
+                    'id' : Math.random().toString(),
```

Comment:
nit: please add a note that this is used as react component key. May be confusing for some one looking at the code for the first time.

Reply:
Added a comment for that